vendor/knative.dev/pkg/metrics/config.go

```diff
@@ -57,8 +59,71 @@ const (
 	defaultPrometheusPort = 9090
 	maxPrometheusPort     = 65535
 	minPrometheusPort     = 1024
+
+	DefaultLogURLTemplate = "http://localhost:8001/api/v1/namespaces/knative-monitoring/services/kibana-logging/proxy/app/kibana#/discover?_a=(query:(match:(kubernetes.labels.knative-dev%2FrevisionUID:(query:'${REVISION_UID}',type:phrase))))"
 )
 
+// ObservabilityConfig contains the configuration defined in the observability ConfigMap.
```

Comment:
I'd think this file changes will be automatically added once your [PR](knative/pkg#786) merged and pkg dependency updated in serving